examples/07_advanced/plot_surface_bids_analysis.py

```diff
@@ -90,55 +112,55 @@
     )
 
     # Fit GLM.
+    # Pass events and all confounds
     first_level_glm.fit(run_imgs=image, events=event[0], confounds=confound[0])
 
-    # Contrast specification
-    design_matrix = first_level_glm.design_matrices_[0]
-    contrast_values = (design_matrix.columns == "language") * 1.0 - (
-        design_matrix.columns == "string"
+    # Compute contrast between 'language' and 'string' events
+    z_scores.append(
+        first_level_glm.compute_contrast(
+            "language-string", stat_type="t", output_type="z_score"
+        )
     )
-    z_scores = first_level_glm.compute_contrast(contrast_values, stat_type="t")
-    z_scores_left.append(z_scores.data.parts["left"])
-    z_scores_right.append(z_scores.data.parts["right"])
 
 
 # %%
 # Group study
 # -----------
 #
-# Individual activation maps have been accumulated
-# in the ``z_score_left`` and ``z_scores_right`` lists respectively.
+# Individual activation maps have been accumulated in the ``z_score``.
 # We can now use them in a group study (one-sample study).
+# We can now pass this to a second level model for a one-sample t-test
+# to fit and compute contrast on.
```

Comment:
```suggestion
# We can now use them in a group study (one-sample study) by passing ``z_score`` to a second level model for a one-sample t-test to fit and compute contrast on.
```

How about a little bit of rephrasing?

Reply:
rephrased more or less the whole passage: LMKWYT